apm-sniffer/apm-sdk-plugin/redisson-3.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/redisson/v3/RedisConnectionMethodInterceptor.java

```diff
@@ -60,6 +63,8 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
 
         StringBuilder dbStatement = new StringBuilder();
         String operationName = "Redisson/";
+        String methodName = "";
```

Comment:
I think the `command` is better than `methodName`, WDYT?

Reply:
I agree with you.